app/jobs/delete_host.rb

```diff
@@ -4,12 +4,23 @@
 class DeleteHost
   include Sidekiq::Worker
 
+  MODELS = [RuleResult, TestResult, PolicyHost].freeze
+
   def perform(message)
+    host_id = message['id']
+    remove_related(host_id)
+  end
+
+  private
+
+  def remove_related(host_id)
+    num_removed = 0
     Host.transaction do
-      Sidekiq.logger.info("Deleting related records for host #{message['id']}")
-      [RuleResult, TestResult, PolicyHost].each do |model|
-        model.where(host_id: message['id']).delete_all
+      Sidekiq.logger.info("Deleting related records for host #{host_id}")
```

Comment:
Should this be audit logged? Why count `num_removed` if it's not used (am I missing something)?

Reply:
See #767

Reply:
Oh! right. :laughing: